README.md

```diff
@@ -0,0 +1,16 @@
+# Zoom Telegram Reminder Bot
+
+This bot notifies a Telegram channel when a new lecture is about to start. Members of the channel can easily register new lectures and view the existing ones. The project has been created primarily for our student Telegram group, but you can use it as well. 
+
+## How to setup
+
+After you've created a Telegram bot, you need a node js server. Clone this repo, install the dependencies, populate the fields in the `.env.example` and build it using `npm run build`. The file referenced with the environment variable `LECTURESPATH` has to contain `[]` to be a valid JSON.
```

Comment:
File must be renamed to .env.
It might also be apt to provide a sample systemd service file, though I would put this as a separate file and link to it.
Eg:
```ini
[Unit]
Description=Telegram Zoom Reminder Bot
After=network.target

[Service]
WorkingDirectory=/opt/TelegramZoomBot/
ExecStart=/usr/bin/node /opt/TelegramZoomBot/dist/index.js
Environment="TELEGRAMTOKEN="
Environment="LECTURESPATH="

[Install]
WantedBy=multi-user.target
```